cvat-ui/src/components/dashboard/header/dashboard-header.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+
+import { Layout, Row, Col, Button, Input } from 'antd';
+
+import './dashboard-header.scss';
+
+const { Header } = Layout;
+const { Search } = Input;
+
+interface DashboardHeaderAction {
+  id: number,
+  name: string,
+  trigger: any,
+}
+
+class DashboardHeader extends Component<any, any> {
+  actions: DashboardHeaderAction[];
+  hostUrl: string;
+
+  constructor(props: any) {
+    super(props);
+
+    this.state = {};
+
+    this.hostUrl = 'http://localhost:7000';
```

Comment:
I see the url in a couple of files. Do we need to have a config component with settings for the backend? Also for the component it is possible to make UI and specify parameters in a dialog (e.g. hostUrl). But by default it should be read from a configuration file during "build" stage